api_spec.yaml

```diff
@@ -1231,11 +1231,217 @@ paths:
                                         type: array
                                         items:
                                             type: string
+    /dashboard/api/tenant:
+        get:
+            tags:
+                - Multitenancy
+            summary: Get tenant info from the core
+            parameters:
+                - name: tenantId
+                  in: query
+                  required: true
+                  schema:
+                      type: string
+                      example: "tenant1"
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
+            responses:
+                200:
+                    description: Success
+                    content:
+                        application/json:
+                            schema:
+                                oneOf:
+                                    - type: object
+                                      properties:
+                                          status:
+                                              type: string
+                                              default: "OK"
+                                          tenant:
+                                              type: object
+                                              properties:
+                                                  tenantId:
+                                                      type: string
+                                                  emailPassword:
+                                                      type: object
+                                                      properties:
+                                                          enabled:
+                                                              type: boolean
+                                                  passwordless:
+                                                      type: object
+                                                      properties:
+                                                          enabled:
+                                                              type: boolean
+                                                  thirdParty:
+                                                      type: object
+                                                      properties:
+                                                          enabled:
+                                                              type: boolean
```

Comment:
can this be simplified? if we are taking the firstFactors as the source of truth

Reply:
Yeah we are using validFirstFactors as the source of truth, but we still might need them for some edge cases

Reply:
Please check if we really need or not. if we really need it, add description as to why we need it